inst/include/sampler.h

```diff
@@ -71,7 +71,7 @@ namespace smc {
         ///Structure used internally for resampling.
         arma::vec dRSWeights;
         ///Structure used internally for resampling.
-        arma::Col<unsigned int> uRSCount;
+        arma::Col<int> uRSCount;
```

Comment:
I wouldn't worry too much, I think most people would curse me more for introducing the convention in the first place. It would be good to try at least to be consistent within the code, but there's absolutely no urgency.